lib/web/appserver.rb

```diff
@@ -29,6 +29,15 @@ def decorate_tags(tags)
   end
 
   #
+  # タグをHTMLで装飾する(除外タグ指定用)
+  #
+  def decorate_exclusion_tags(tags)
+    tags.sort.map do |tag|
+      %!<span class="tag label label-#{Command::Tag.get_color(tag)}" data-exclusion-tag="#{escape_html(tag)}">^tag:#{escape_html(tag)}</span>!
```

Comment:
__[rubocop]__
* Metrics/LineLength: Line is too long. [142/140] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)